src/main/java/com/blackducksoftware/integration/hub/notification/NotificationResults.java

```diff
@@ -24,35 +24,29 @@
 package com.blackducksoftware.integration.hub.notification;
 
 import java.util.List;
-import java.util.SortedSet;
 
 import org.apache.commons.lang3.builder.RecursiveToStringStyle;
 import org.apache.commons.lang3.builder.ReflectionToStringBuilder;
 
-public class NotificationResults {
-    private final SortedSet<NotificationContentItem> notificationContentItems;
+import com.blackducksoftware.integration.hub.api.view.CommonNotificationState;
+import com.blackducksoftware.integration.hub.service.bucket.HubBucket;
 
-    private final List<Exception> exceptions;
+public class NotificationResults {
+    private final List<CommonNotificationState> notificationContentItems;
+    private final HubBucket hubBucket;
 
-    public NotificationResults(final SortedSet<NotificationContentItem> notificationContentItems, final List<Exception> exceptions) {
+    public NotificationResults(final List<CommonNotificationState> notificationContentItems, final HubBucket hubBucket) {
```

Comment:
Can we make this a generic Collection instead of a List?

Reply:
We certainly can.